source/agora/stats/EndpointReqestStats.d

```diff
@@ -0,0 +1,16 @@
+module agora.stats.EndpointReqestStats;
+
+import agora.stats.Stats;
+
+public struct EndpointRequestStatsLabel
+{
+    string endpoint;
+    string protocol;
+}
```

Comment:
Question: Can we log the originating IP ?
That might create too many dimensions though, so another possibility is log the peer only if the request is authenticated (comes from a validator).

Reply:
we can log it(as long as I can get the IP somehow), it only increases the dimension number by 1